src/libraries/System.Net.Requests/src/System/Net/FtpWebRequest.cs

```diff
@@ -220,6 +220,7 @@ public sealed class FtpWebRequest : WebRequest
         private LazyAsyncResult? _readAsyncResult;
         private LazyAsyncResult? _requestCompleteAsyncResult;
 
+        // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Unit test dummy credentials.")]
```

Comment:
Can you fix this comment, this is not a unit test credentials. I guess it is the standard credentials for anonymous FTP?

Reply:
Is it possible to put `anonymous` and `anonymous@` into the json file so you don't need to add this comment? If so I think that would be better.